src/StatsdClient/StatsdClient.csproj

```
@@ -4,13 +4,16 @@
     <PackageId>DogStatsD-CSharp-Client</PackageId>
     <Description>A DogStatsD client for C#. DogStatsD is an extension of the StatsD metric server for use with Datadog. For more information visit http://datadoghq.com.</Description>
     <Authors>Datadog</Authors>
-    <TargetFrameworks>netstandard1.3;netstandard1.6;netcoreapp1.1;netcoreapp2.0;net451;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net451</TargetFrameworks>
     <Version>3.3.0</Version>
     <PackageLicenseUrl>https://github.com/DataDog/dogstatsd-csharp-client/blob/master/MIT-LICENCE.md</PackageLicenseUrl>
     <PackageProjectUrl>https://github.com/DataDog/dogstatsd-csharp-client</PackageProjectUrl>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3' OR '$(TargetFramework)' == 'netstandard1.6' OR '$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'netcoreapp2.0'">
+  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
+    <DefineConstants>OS_WINDOWS</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' != 'net451'">
```

If we make the change I suggested above, we need to change this, too.

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' != 'net451' && '$(TargetFramework)' != 'net461'">
```